build/rust/config.gni

```diff
@@ -30,6 +30,13 @@ if (is_win) {
       rust_flags += " -C link-arg=--sysroot=$target_sysroot "
       rust_flags += " -C link-arg=-Wl,-rpath=\$ORIGIN "
     }
+  } else if (current_cpu == "arm64") {
+    rustc_target = "aarch64-unknown-linux-gnu"
+    rustc_toolchain = "arm64"
+    if (use_sysroot) {
+      rust_flags += " -C link-arg=--sysroot=$target_sysroot "
+      rust_flags += " -C link-arg=-Wl,-rpath=\$ORIGIN "
+    }
```

Comment:
I think we also need a `-C linker=xxxxxxx` argument as per [rustc docs](https://doc.rust-lang.org/rustc/codegen-options/index.html#linker)